perception/object_merger/src/object_association_merger/node.cpp

```diff
@@ -69,9 +69,10 @@ ObjectAssociationMergerNode::ObjectAssociationMergerNode(const rclcpp::NodeOptio
   remove_overlapped_unknown_objects_ =
     declare_parameter<bool>("remove_overlapped_unknown_objects", true);
   overlapped_judge_param_.precision_threshold =
-    declare_parameter<double>("precision_threshold_to_judge_overlapped", 0.4);
+    declare_parameter<double>("precision_threshold_to_judge_overlapped");
   overlapped_judge_param_.recall_threshold =
     declare_parameter<double>("recall_threshold_to_judge_overlapped", 0.5);
+  distance_threshold_ = declare_parameter<double>("distance_threshold");
```

Comment:
Since this variable is only an overlap judgment for UNKNOWN, I want to separate the namespace.

Reply:
I changed distance_threshold to distance_threshold_list, and it enables to separate distance_threshold by object class.